sfge/src/main/java/com/salesforce/rules/OccurrenceInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.salesforce.rules;
+
+import com.salesforce.config.UserFacingMessages;
+
+/** Internal representation of an occurrence info. Originally from MMSL Info Utils. */
```

Comment:
Just the first sentence is fine. No need for a code-level comment to indicate where it came from. (Though this kind of refactor is a good thing to call out with a comment on the PR itself.)